src/index.html

```diff
@@ -13,6 +13,38 @@
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-sun calendar--month-length-31">
```

Comment:
```suggestion
    <div class="calendar calendar--start-day--sun calendar--month-length--31">
```
let's fix modifiers class naming according to BEM naming convention

Reply:
![image](https://github.com/mate-academy/layout_calendar/assets/115177224/13e297d5-e82d-4cbc-aa14-9311bea79d11)
I can`t change this because of tests(